templates/taxi/index.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% block content %}
+    <ul>
+        <li>Number of cars: {{ num_cars }}</li>
+        <li>Number of drivers: {{ num_drivers }}</li>
+        <li>Number of manufacturers: {{ num_drivers }}</li>
```

Comment:
It seems like you have a typo here. You're displaying the number of drivers as the number of manufacturers. You should probably use a different variable, like `num_manufacturers`, to display the correct information.